packages/big-design/src/components/Table/Row/Row.tsx

```diff
@@ -28,18 +30,20 @@ const InternalRow = <T extends TableItem>({
     }
   };
 
+  const label = isSelected ? `Deslect item number ${itemIndex + 1}` : `Select item number ${itemIndex + 1}`;
```

Comment:
Typo:
```suggestion
  const label = isSelected ? `Deselect item number ${itemIndex + 1}` : `Select item number ${itemIndex + 1}`;
```

🍹What about renaming to something like "Deselect row {number}"?

Reply:
Do we need the item index? The Row already has the context of the `item`, is there any benefits of having the index as part of the label?

Reply:
This was a suggestion from @chanceaclark as an option for an accessible label, but the label for the checkbox was arbitrarily chosen and can be changed. Let's discuss offline 🌮